apps/ledger-live-mobile/src/icons/AppTree.tsx

```diff
@@ -21,51 +21,16 @@ export default function AppTree({ color, icon, app }: Props) {
       </Flex>
       <Svg width="163" height="50" viewBox="0 0 163 50" fill="none">
         <G opacity="0.25">
-          <Image
-            x="141"
-            y="27"
-            width="22"
-            height="22"
-            rx="11"
-            ry="11"
```

Comment:
these props don't exist and it was not working anyway (I compared with develop)

Reply:
for info this is how the icon looks without `rx` and `ry`
![IMG_AE60E2679FED-1](https://github.com/LedgerHQ/ledger-live/assets/91890529/b265b770-1676-4c29-a7d6-19ea60d9529f)